boards/xtensa/waveshare_esp32s3_touch_lcd/waveshare_esp32s3_touch_lcd_defconfig

```diff
@@ -0,0 +1,13 @@
+# Copyright (c) 2024 Joel Guittet
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_SOC_SERIES_ESP32S3=y
+CONFIG_BOARD_WAVESHARE_ESP32S3_TOUCH_LCD=y
+CONFIG_MAIN_STACK_SIZE=2048
+CONFIG_GPIO=y
+CONFIG_REGULATOR=y
+CONFIG_CONSOLE=y
+CONFIG_PWM=y
```

Comment:
I think that normally subsystems options apart from the console one are left up to the application, @kartben?

Reply:
I'm curious of the strategy here. Also curious about why there is _defconfig and Kconfig.defconfig. Sometimes they overlap and I still not understand the usage of both files.

Reply:
Instead of LVGL maybe "if DISPLAY" ? just in case somebody write a display application without LVGL support
Can be the following in Kconfig.defconfig:
```
config PWM
        default y if DISPLAY
```

Reply:
yeah that works, note that I put a comment on https://github.com/zephyrproject-rtos/zephyr/pull/67012#discussion_r1444701046 about PWM usage on that driver, I think that the pwm backlight implementation could use a more generic implementation, let's follow up on that PR but the change will certainly reflect back on this one